lib/libvfio-user.c

```diff
@@ -1455,9 +1480,10 @@ vfu_setup_region(vfu_ctx_t *vfu_ctx, int region_idx, size_t size,
     switch(region_idx) {
     case VFU_PCI_DEV_BAR0_REGION_IDX ... VFU_PCI_DEV_VGA_REGION_IDX:
         // Validate the config region provided.
-        if (region_idx == VFU_PCI_DEV_CFG_REGION_IDX &&
-            !is_valid_pci_config_space_region(flags, size)) {
+        if (region_idx == VFU_PCI_DEV_CFG_REGION_IDX) {
+            if (flags == VFU_REG_FLAG_RW) {
```

Comment:
what is this really testing for? Not using & for flags looks odd at least?

Reply:
sorry that shoud be `!=`

Reply:
> what is this really testing for? Not using & for flags looks odd at least?

sorry that shoud be !=